doc/source/computation.rst

```diff
@@ -335,32 +379,32 @@ This keyword is available in other rolling functions as well.
 
 .. ipython:: python
 
-   pd.rolling_window(ser, 5, 'boxcar')
+   ser.rolling(window=5, win_type='boxcar').mean()
```

Comment:
I think the ``win_type='boxcar'`` is not needed here. This is already explained in the section just above this one (so it is already shown that this gives the same result as the default without `win_type`), and would make this example more focused on difference between center=True/False